scripts/development/m3_stack/README.md

```diff
@@ -0,0 +1,28 @@
+# Local Development
```

Comment:
mind linking a new section in https://github.com/m3db/m3/blob/master/DEVELOPER.md to this page